bin/npm.cmd

```diff
@@ -1,17 +1,6 @@
 @echo off
 setlocal EnableDelayedExpansion
 
-:: Try to find git.exe in path
```

Comment:
We didn't understand why it was needed as part of `npm.cmd`... Made more sense in `apm.cmd`. Assumed it was copied and pasted. Am I wrong?